.goreleaser.yml

```
@@ -110,16 +110,14 @@ brews:
       system "#{bin}/terraform-ls --version"
 
 publishers:
-  - name: "hc-releases"
+  - name: upload
```

```suggestion
  - name: "hc-releases"
```
I think in the context of GoRelaser config, the original name is still relevant?